applications/GeoMechanicsApplication/tests/three_dimensional_piping/MaterialParameters.json

```diff
@@ -0,0 +1,97 @@
+{
+    "properties": [
+        {
+            "model_part_name": "PorousDomain.Soil_1",
+            "properties_id": 1,
+            "Material": {
+                "constitutive_law": {
+                    "name": "LinearElasticK03DLaw"
```

Comment:
Can this now become GeoLinearElasticLaw for all 3 materials? The K0 thing has no meaning here.